fastlane/Fastfile

```diff
@@ -829,15 +827,20 @@ end
       type: "appstore",
       team_id: get_required_env("EXT_EXPORT_TEAM_ID"),
       readonly: options[:readonly] || is_ci,
-      app_identifier: [
-        "com.codality.NotationalFlow",
-        "com.codality.NotationalFlow.Share",
-        "com.codality.NotationalFlow.Widgets",
-      ],
+      app_identifier: simplenote_app_identifiers(root_bundle_id: APP_STORE_BUNDLE_IDENTIFIER),
       template_name: "NotationalFlow Keychain Access (Distribution)",
     )
   end
 
+  def simplenote_app_identifiers(root_bundle_id:)
+    [
+      root_bundle_id,
+      "#{root_bundle_id}.Share",
+      "#{root_bundle_id}.Widgets",
+      "#{root_bundle_id}.Intents"
+    ]
+  end
```

Comment:
I decided to DRY the code defining the bundle identifiers a bit because of their constant structure. It made it simpler to add the "Intents" one and be sure I didn't forget it from one build configuration entry.